src/pybind/mgr/cephadm/module.py

```diff
@@ -407,6 +407,12 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
             default=False,
             desc='Log all refresh metadata. Includes daemon, device, and host info collected regularly. Only has effect if logging at debug level'
         ),
+        Option(
+            'certificate_check_debug_mode',
+            type='bool',
+            default=False,
+            desc='FOR TESTING ONLY: This flag forces the certificate check instead of waiting for certificate_check_period.',
+        ),
```

Comment:
I think you can add `level=OptionLevel.DEV` or something to this and it hides it

Reply:
Ohh, that's a really nice option that I didn’t know about! Thanks! :+1: